spec/samples/inline_trackers.js

```diff
@@ -0,0 +1,93 @@
+export const inlineTrackers = `
+<VAST version="2.1">
```

Comment:
why not just add the tracking events to an existing test VAST? like `inlineLinear`?